scripts/copy-monaco-assets.mjs

```diff
@@ -0,0 +1,4 @@
+import shell from 'shelljs'
+
+// copy workers to public
+shell.cp('-fr', './node_modules/monaco-editor-workers/dist/', './public/monaco-editor-workers');
```

Comment:
You actually only require those three files:
```shell
monaco-editor-workers/dist/index.js
monaco-editor-workers/dist/workers/editorWorker-es.js
monaco-editor-workers/dist/workers/editorWorker-iife.js
```
All other workers are for languages you don't need (e.g. TypeScript)